src/util/gcode-preview.ts

```diff
@@ -1,5 +1,6 @@
 import { ArcMove, Move, Point, Rotation } from '@/store/gcodePreview/types'
 
+// eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
Instead replace `Function` with `(item: any, index: number, array: any[]) => number`?

Reply:
Done, and made it generic so one doesn't need to use `any`!